### ISSUE TITLE
Scenario editor

### ISSUE THREAD

**Issue by smithhmark** [smithhmark/sub-joust#8]
It would be very nice to have a visual simulation editor. 

Some kind of plotter analog, with options to place a contact, then script the contact's maneuvers.
Also, the ability to describe vessel classes, especially in terms of the performance envelope and related "noisiness".

**Comment by smithhmark** [comment]
dependent on #2
